templates/cli/lib/sdks.js.twig

```diff
@@ -7,7 +7,24 @@ const questionGetEndpoint = [
     type: "input",
     name: "endpoint",
     message: "Enter the endpoint of your {{ spec.title|caseUcfirst }} server",
-    default: "http://localhost/v1"
+    default: "http://localhost/v1",
+    validate: async (value) => {
+      if(!value) {
+        return 'Please provide valid endpoint';
+      }
+      let client = new Client().setEndpoint(value);
+      try {
+        let response = await client.call('GET', '/health/version');
+        if(response.version) {
+          return true;
+        } else {
+          return 'Not a valid endpoint or Appwrite server is not running as expected';
+        }
+      } catch (error) {
+        return 'Not a valid endpoint or Appwrite server is not running as expected';
+      }
+      return true;
+    }
```

Comment:
```suggestion
    validate(value) {
      if (!value) {
        return "Please enter a valid endpoint.";
      }

      let client = new Client().setEndpoint(value);
      
      try {
        let response = await client.call('get', '/health/version');
        if (response.version) {
          return true;
        } else {
          throw new Error();
        }
      } catch (error) {
        return "Invalid endpoint or your Appwrite server is not running as expected.";
      }
    }
```

Reply:
Why this? Arrow function works, Also, we need this to be async?

Reply:
Updated this, removed arrow function for consistency